### ISSUE TITLE
Keras version

### ISSUE THREAD

**Issue by agatheminaro** [cyprienruffino/CTCModel#20]
Hi,

Thanks for this beautiful work. Just a question : what is the version of keras used for this, please ?

Agathe

**Comment by cyprienruffino** [comment]
Hi ! 
It is supposed to work with the latest versions of Keras. If you encounter compatibility issues with any version, can you specify which one, so that I can check on the issue ?

Thanks !

**Comment by agatheminaro** [comment]
Yes I had to make some small changes on the packages import by replacing : 
- `from keras.engine import Model` by `from keras.engine.training import Model`
- `from keras.utils import Sequence, GeneratorEnqueuer, OrderedEnqueuer` by `from tensorflow.keras.utils import Sequence, GeneratorEnqueuer, OrderedEnqueuer`
- `from keras.optimizers import Adam` by `from tensorflow.keras.optimizers import Adam`

**Comment by agatheminaro** [comment]
Also the dropbox link in the _example.py_ file does not work anymore.

**Comment by cyprienruffino** [comment]
Thanks ! I'll look into that immediately. I've updated the dropbox link, it should work now. 
Which version of Keras and TensorFlow are you using ?

**Comment by agatheminaro** [comment]
I'm using the version 2.6.0 of Keras and Tensorflow.

**Comment by xiaozh0202** [comment]
@agatheminaro I want to know that which one you are using？Tensorflow or Tensorflow-gpu？
and @cyprienruffino What tf and Keras versions did you use when developing this extension?  Because I can only run with tf-CPU now, but the speed is too slow, but the various versions of TF-GPU are not successful.

**Comment by Lauren001** [comment]
> @agatheminaro I want to know that which one you are using？Tensorflow or Tensorflow-gpu？ and @cyprienruffino What tf and Keras versions did you use when developing this extension? Because I can only run with tf-CPU now, but the speed is too slow, but the various versions of TF-GPU are not successful.
May I ask what the versions of tensorflow and keras you use are？
cause the version i use have the problem 
module 'tensorflow.python.keras.utils.tf_utils' has no attribute 'to_numpy_or_python_type'

**Comment by Lauren001** [comment]
> @agatheminaro I want to know that which one you are using？Tensorflow or Tensorflow-gpu？ and @cyprienruffino What tf and Keras versions did you use when developing this extension? Because I can only run with tf-CPU now, but the speed is too slow, but the various versions of TF-GPU are not successful.

May I ask what the versions of tensorflow and keras you use are？
cause the version i use have the problem
module 'tensorflow.python.keras.utils.tf_utils' has no attribute 'to_numpy_or_python_type'
